locales/en-US/messages.properties

```diff
@@ -280,10 +280,11 @@ contributeReason1=Your work impacts users all over the world
 contributeReason2=Gain valuable experience and mentorship
 contributeReason3=Support Mozilla's mission of an open internet for all
 contributeReason4=It feels good!
-
-getInvolvedDescription1=Thimble is an <a href="https://en.wikipedia.org/wiki/Open-source_software">Open Source</a> project created by the <a href="https://www.mozilla.org/foundation/">Mozilla Foundation</a> in partnership with <a href="http://cdot.senecacollege.ca/">CDOT at Seneca College</a>. Since the beginning, contributors from all over the world have played an important role in making it a success.
+wikipediaOpenSourceURL=https://en.wikipedia.org/wiki/Open-source_software
+mozillaFoundationURL=https://www.mozilla.org/foundation/
```

Comment:
Exactly. I know it’s a bit confusing (sorry, hard to explain), but you nailed it, @flukeout